come-back.php

```diff
@@ -27,7 +27,7 @@
  */
 const COME_BACK_VERSION = '1.3.4';
 
-require_once 'vendor/autoload.php';
+require_once __DIR__ . ‘/vendor/autoload.php’;
```

Comment:
The character `‘` is actually invalid. This should be replaced with `'`.

> The character U+2018 "‘" could be confused with the character U+0060 "`", which is more common in source code

Reply:
Hey @sanzeeb3 , I have corrected the issue. Let me know if it's good now.